release-docs/0.10/CHANGES.0.10.md

```diff
@@ -0,0 +1,215 @@
+| [FALCON-1763](https://issues.apache.org/jira/browse/FALCON-1763) | Create a spark execution engine for Falcon |  Major | . | Venkat Ranganathan | Peeyush Bishnoi |
+| [FALCON-1627](https://issues.apache.org/jira/browse/FALCON-1627) | Provider integration with Azure Data Factory pipelines |  Major | . | Venkat Ranganathan | Ying Zheng |
+| [FALCON-887](https://issues.apache.org/jira/browse/FALCON-887) | Support for multiple lib paths in falcon process |  Minor | process | Akshay Goyal | Sowmya Ramesh |
+
+
+### IMPROVEMENTS:
+
+| JIRA | Summary | Priority | Component | Reporter | Contributor |
+|:---- |:---- | :--- |:---- |:---- |:---- |
+| [FALCON-2020](https://issues.apache.org/jira/browse/FALCON-2020) | Changes in Graphite Notification Plugin |  Major | . | Praveen Adlakha | Praveen Adlakha |
+| [FALCON-2008](https://issues.apache.org/jira/browse/FALCON-2008) | Add documentation for Graphite Notification Plugin |  Major | . | Praveen Adlakha | Praveen Adlakha |
+| [FALCON-1981](https://issues.apache.org/jira/browse/FALCON-1981) | Remove runtime superfluous jar dependencies - pom.xml cleanup |  Major | build-tools | Venkatesan Ramachandran | Venkatesan Ramachandran |
+| [FALCON-1926](https://issues.apache.org/jira/browse/FALCON-1926) | Filter out effectively non-falcon related JMS messages from Oozie |  Major | messaging | Venkatesan Ramachandran | Venkatesan Ramachandran |
+| [FALCON-1899](https://issues.apache.org/jira/browse/FALCON-1899) | Create examples artifact module in Falcon |  Major | . | Peeyush Bishnoi | Peeyush Bishnoi |
+| [FALCON-1896](https://issues.apache.org/jira/browse/FALCON-1896) | Failure in Falcon build in distro module |  Major | . | Praveen Adlakha | Praveen Adlakha |
+| [FALCON-1895](https://issues.apache.org/jira/browse/FALCON-1895) | Refactoring of FalconCLI and FalconClient |  Major | client | Praveen Adlakha | Praveen Adlakha |
+| [FALCON-1841](https://issues.apache.org/jira/browse/FALCON-1841) | Grouping test in falcon for running nightly regression |  Major | regression | Pragya Mittal | Pragya Mittal |
+| [FALCON-1835](https://issues.apache.org/jira/browse/FALCON-1835) | Falcon should do coord rerun rather than workflow rerun to ensure concurrency |  Major | . | Pallavi Rao | Pallavi Rao |
+| [FALCON-1818](https://issues.apache.org/jira/browse/FALCON-1818) | Minor doc update for tar package locations after FALCON-1751 |  Minor | . | Deepak Barr | Deepak Barr |
+| [FALCON-1774](https://issues.apache.org/jira/browse/FALCON-1774) | Better message for api not allowed on server |  Major | . | Sanjeev T | Praveen Adlakha |
+| [FALCON-1751](https://issues.apache.org/jira/browse/FALCON-1751) | Support assembly:single mojo |  Minor | . | ruoyu wang | ruoyu wang |
+| [FALCON-1584](https://issues.apache.org/jira/browse/FALCON-1584) | Falcon allows invalid hadoop queue name for schedulable feed entities |  Major | . | Venkatesan Ramachandran | Venkatesan Ramachandran |
+
+
+### BUG FIXES:
```

Comment:
Some important features are listed as bug fixes in the generated Change log. I will go ahead and update the 0.10 Jiras to properly reflect compatibility, improvement, feature etc in an Apache Yetus compatible way. Then we need to re-generate the change-log. 

The same needs to be done for all other releases or we lose some important information on backwards compatibility, features, improvements etc.